asm/melee/cm/camera.s

```diff
@@ -9121,7 +9121,7 @@ func_80030BBC:
 /* 80030BF0 0002D7D0  83 E5 00 28 */	lwz r31, 0x28(r5)
 /* 80030BF4 0002D7D4  38 A1 00 28 */	addi r5, r1, 0x28
 /* 80030BF8 0002D7D8  38 7F 00 00 */	addi r3, r31, 0
-/* 80030BFC 0002D7DC  4B FD D6 15 */	bl func_8000E210
+/* 80030BFC 0002D7DC  4B FD D6 15 */	bl lbvector_project_world_to_screen_coord
```

Comment:
Let's stick to shorter names and convention of something like:
lbvector_WorldToScreen